src/styles/globals.css

```diff
@@ -36,3 +36,16 @@ body {
 .error-message {
   @apply flex items-center justify-center text-center text-gray-300 relative top-0 left-0 bottom-0 right-0 h-screen flex-col;
 }
+
+
+
+/* Hide scrollbar for Chrome, Safari and Opera */
+.example::-webkit-scrollbar {
+  display: none;
+}
+
+/* Hide scrollbar for IE, Edge and Firefox */
+.example {
+  -ms-overflow-style: none;  /* IE and Edge */
+  scrollbar-width: none;  /* Firefox */
+}
```

Comment:
We should probably not name this class `example`. Maybe something like `hide-scrollbar`?

Reply:
Updated to hide-scrollbar